src/event-stream/event-server.ts

```diff
@@ -817,6 +817,7 @@ export async function startEventServer(opts: {
         res.status(200).json({ result: 'ok' });
       } catch (error) {
         logError(`error processing core-node /new_block: ${error}`, error);
+        logError(`Event-replay process failed due to the aforementioned error`);
```

Comment:
These errors could happen also if there wasn't an event replay in progress, so I don't think these messages should be printed here.

IIRC, there's an `eventReplay` property in the `index.ts` file that shows whether or not we're actually performing one, so I think this logic should be handled over there.

Reply:
Thanks for the help on this @rafaelcr 

I went through the documentation for [Event-Replay](https://github.com/hirosystems/stacks-blockchain-api#event-replay) and noticed that event-replay initiates on `export-events` or `import-events` commands are given. Therefore, I've logged the error in the catch. Can you check it once and let me know if that looks good to you? 

cc: @CharlieC3